blaze/interactive.py

```diff
@@ -41,19 +41,27 @@
     not_an_iterator.append(pymongo.database.Database)
 
 
-class InteractiveSymbol(Symbol):
-    """Interactive data.
+class _Data(Symbol):
```

Comment:
Yes, that's exactly the ultimate intent, and in version 0.11 we'll remove `InteractiveSymbol` and rename `_Data` to `Data`.  I made note of this in the whatsnew document.